Source/dotnet-storedprocsgen/Database/StoredProcedureGenerator.cs

```diff
@@ -82,6 +86,14 @@ public string GetBody(StoredProcedureKind procedureKind, List<ColumnInfo> column
                 result.AppendLine(GetSearchRawData(columns, sortByColumns, searchColumns));
                 result.AppendLine(GetSearchFinalSelect());
             }
+            else if (procedureKind == StoredProcedureKind.GetById) {
+                var primaryColumn = columns.First(c => c.PrimaryKeyColumnPosition == 1);
+                result.Append("SELECT ");
+                result.Append(GetInsertStatementColumnsList(columns));
+                result.AppendLine($" FROM [{firstColumn.SchemaName}].[{firstColumn.TableName}]");
+                result.AppendLine("WHERE");
+                result.AppendLine(GetUpdateStatementWhere(columns));
```

Comment:
This is not quire right since it includes version column

Reply:
I think the issue is still there UpdateStatementWhere will contain row version, and for get by id we should only use primary key columns.  Also insert column list will exclude auto generated columns, but for select we need all the columns.

Reply:
I think the issue is still there UpdateStatementWhere will contain row version, and for get by id we should only use primary key columns.  Also insert column list will exclude auto generated columns, but for select we need all the columns.